packages/tiny-agents/package.json

```diff
@@ -34,7 +34,8 @@
 		"prepare": "pnpm run build",
 		"test": "vitest run",
 		"check": "tsc",
-		"cli": "tsx src/cli.ts"
+		"cli": "tsx src/cli.ts",
+		"cli:watch": "tsx watch src/cli.ts"
```

Comment:
I'm running with `pnpm cli:watch serve ./src/agents/julien-c/local-coder/`

when I change code in `packages/mcp-client/src/Agent.ts`, tsx build does NOT get triggered and I'd beed to manually `cd packages/mcp-client && pnpm build`. Am I doing something wrong or this `cli:watch` script could be better by watching its dependencies as well?

cc: @coyotte508

Reply:
yes can you try adding `--include ../mcp-client/src` to this command then @mishig25?